sdks/ios/Makefile

```diff
@@ -77,8 +77,8 @@ TEST_ASSEMBLIES = $(BCL_DIR)/mscorlib.dll \
 	test-runner.exe
 
 build-ios-sim-%: appbuilder.exe test-runner.exe runtime/runtime $($*_ASSEMBLIES)
-	mono appbuilder.exe --target ios-sim64 --mono-sdkdir $(PWD)/../out --appdir $(PWD)/bin/ios-sim/test-$*.app --runtimedir $(PWD)/runtime --builddir obj/ios-sim/test-$*.app --sysroot $(SYSROOT) --signing-identity - --bundle-executable test-$* --bundle-identifier com.xamarin.mono.ios.test-$* --bundle-name test-$* $(patsubst %,-r %,$(TEST_ASSEMBLIES) $($*_ASSEMBLIES))
-	mkdir -p $(PWD)/bin/ios-sim/test-$*.app
```

Comment:
make -C is supposed to change the working directory:
"Change to directory dir before reading the makefiles or doing anything else."

Reply:
not really. you can use `$(CURDIR)` instead:
```shell
$ pwd
/tmp
$ mkdir -p deep/dir/path
$ cat > deep/dir/path/Makefile
all:
	echo $(PWD)
	echo $(CURDIR)
$ make -C deep/dir/path
echo /tmp
/tmp
echo /private/tmp/deep/dir/path
/private/tmp/deep/dir/path
```

Reply:
PWD is an environment variable, and it seems it's not updated automatically when the current directory changes.

`$(CURDIR)` is a make variable which reflects the current directory and could be used instead: `Set to the absolute pathname of the current working directory (after all -C options are processed, if any).`

Reply:
Yeah, I didn't try whether passing the relative dirs to `appbuilder.exe` works (since `make` changes the working dir) but `abspath` works and fixes the issue :)